CHANGELOG

```diff
@@ -1,5 +1,11 @@
 = Heart Check - Changelog
 
+== Version 1.7.0 :: 2020-03-10
```

Comment:
What about increasing the major to 2.0.0 since there are break changes (it will not support old rubies)?

Reply:
make sense...